recipe_scrapers/_abstract.py

```diff
@@ -63,54 +63,55 @@ def __init__(
 
     @classmethod
     def host(cls) -> str:
-        """get the host of the url, so we can use the correct scraper"""
+        """Host domain of the recipe URL"""
         raise NotImplementedError("This should be implemented.")
 
     def canonical_url(self):
+        """Canonical or original URL of the recipe"""
         canonical_link = self.soup.find("link", {"rel": "canonical", "href": True})
         if canonical_link:
             return urljoin(self.url, canonical_link["href"])
         return self.url
 
     def title(self):
+        """Title of the recipe."""
         raise NotImplementedError("This should be implemented.")
 
     def category(self):
+        """Category of the recipe."""
         raise NotImplementedError("This should be implemented.")
 
     def total_time(self):
-        """total time it takes to preparate and cook the recipe in minutes"""
+        """Total time needed to prepare and cook the recipe in minutes."""
         raise NotImplementedError("This should be implemented.")
 
     def cook_time(self):
-        """cook time of the recipe in minutes"""
+        """Cooking time in minutes."""
         raise NotImplementedError("This should be implemented.")
 
     def prep_time(self):
-        """preparation time of the recipe in minutes"""
+        """Preparation time in minutes."""
         raise NotImplementedError("This should be implemented.")
 
     def yields(self):
-        """The number of servings or items in the recipe"""
+        """Total servings or items in the recipe."""
         raise NotImplementedError("This should be implemented.")
 
     def image(self):
+        """An image URL for the recipe."""
         raise NotImplementedError("This should be implemented.")
 
     def nutrients(self):
+        """Nutrients of the recipe."""
         raise NotImplementedError("This should be implemented.")
 
     def language(self):
-        """
-        Human language the recipe is written in.
-
-        May be overridden by individual scrapers.
-        """
+        """Language the recipe is written in."""
         candidate_languages = OrderedDict()
         html = self.soup.find("html", {"lang": True})
         candidate_languages[html.get("lang")] = True
 
-        # Deprecated: check for a meta http-equiv header
+        # Deprecated: Check for a meta http-equiv header
```

Comment:
Super-micro-nitpick:

```suggestion
        # Deprecated: check for a meta http-equiv header
```

(can remain as-is - same thing about the capital letters)